### ISSUE TITLE
Pretrained models' performance doesn't match the result

### ISSUE THREAD

**Issue by ykxixi** [antoyang/TubeDETR#7]
Hi, I download the checkpoints pretrained on HC-STVG2.0, but the result is: viou：0.3555, viou@0.3: 0.5675, viou@0.5: 0.3000. I also find the loss is larger than 25, and the loss of the 0 epoch is almost 58. I have change the stride and resolution to match the checkpoints' training configuration. Did I miss something?

**Comment by antoyang** [comment]
Hi, I have just reevaluated the checkpoint for HC-STVG2.0 res=352 stride=4 with the command 'python -m torch.distributed.launch --nproc_per_node=8 --use_env main.py --eval --ema --combine_datasets=hcstvg --combine_datasets_val=hcstvg --dataset_config config/hcstvg.json --v2 --output-dir=hcstvgv2k4res352 --load <PATH_TO_CKPT> --stride=4 --resolution=352' and obtained the reported performance. I have put the log_stats.json here for more details: https://drive.google.com/file/d/1lACP-bBb6mwpv00et7tqEdsWq633QSMv/view?usp=sharing.
